tests/SprykerSdkTest/Integrator/IntegratorFacadeTest.php

```diff
@@ -252,6 +252,29 @@ public function testRunInstallationUnwireGlueRelaitonship(): void
         $this->assertSame(trim(file_get_contents($testFilePath)), trim(file_get_contents($classPath)));
     }
 
+    /**
+     * @return void
+     */
+    public function testRunInstallationGlossary(): void
+    {
+        // Arrange
+        $ioAdapter = $this->buildSymfonyConsoleInputOutputAdapter();
+
+        // Act
+        $this->createIntegratorFacade()
+            ->runInstallation($this->getModuleList('TestIntegratorGlossary'), $ioAdapter, false);
+
+        // Assert
+        $existingGlossaryFilePath = './tests/_data/project_mock/data/import/common/common/glossary.csv';
```

Comment:
```suggestion
        $projectGlossaryFilePath = './tests/_data/project_mock/data/import/common/common/glossary.csv';
```
might be more speaking as this is about the project one.